web/styles/popovers.css

```diff
@@ -856,3 +856,24 @@ ul {
         column-count: 1;
     }
 }
+
+#send_later_popover {
+    & hr {
+        margin: 5px 0;
+    }
+
+    .send_later_popover_header {
+        text-align: center;
+        font-weight: bold;
+
+        .fa-chevron-right {
+            color: hsl(240deg 96% 68%);
+            margin-right: 3px;
+        }
```

Comment:
Is this CSS stale, since we don't have a chevron there anymore?

Reply:
Yeah, sorry, forgot to remove it.